docs/source/tutorials/GetMetadataNCBI.md

```diff
@@ -0,0 +1,56 @@
+# Get Metadata from NCBI 
+
+Extract metadata from NCBI. 
+
+Tool: [NCBImeta](https://github.com/ktmeaton/NCBImeta)
+
+Example local installation
+```
+conda create -n ncbimeta -c bioconda ncbimeta
```

Comment:
Add a comment here that this is already installed, so mention how to activate it